app/tournament/player.js

```diff
@@ -0,0 +1,23 @@
+export class Player {
+    constructor(name, rank) {
+        this.name = name;
+        this.rank = rank;
+    }
+}
+
+export function validatePlayer(data) {
+    var validationResult = {};
+
+    validationResult.error = validateName(data.name) + validateRank(data.rank);
```

Comment:
Don't concatenate, use a list. Let the UI to make the decision of how to show data.